content/nextflow-on-aws-batch/30_batch/40_job_queue.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Job Queue"
+chapter: false
+weight: 40
+---
+
+## Create Job Queue
+
+Two queues need to be created. Both are created via the consol.
+![](/images/nextflow-on-aws-batch/batch/5_queue_workflow_0.png)
+
+To create both queues we choose a name (**workflow-queue** / **job-queue**) a priority of 1 and the map them to the correct compute environment.
```

Comment:
I don't think this is clear enough. I assumed that the workflow-queue connects to the od-ce and the job-queue connects to the spot-ce

Reply:
I adjusted it a tad